lib/ip_wrangler/main.rb

```diff
@@ -60,7 +56,7 @@ def check_released_resource(released_resource)
   sandbox do
     $nat.get_nat_ports.map { |nat_port|
       puts nat_port[:public_ip]
-      nat_port[:public_ip] = $config[:ext_ip]
+      nat_port[:public_ip] = $config['ext_ip']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.